### ISSUE TITLE
LSP not working: very specific case

### ISSUE THREAD

**Issue by Jrcarand** [spyder-ide/spyder#16556]
Hello (concise writing at bottom)

I wanted to try the standalone installer, but did not do enought research before intalling it and noticed it does not come with any form of support to add packages. Specifically, I wrote several packages that I use on a daily basis that I install via pip.

So, I did what was recommended and went for miniconda and setup everything, and everything was fine.

However, being this a completely new PC, I decided to debloat the installation as much as possible, so I wanted to just have miniconda.

So I proceeded to uninstall Spyder, and went through conda to install spyder and all required dependencies. However, after this the LSP was not working and just trying to restart. As it was working before I do not think this is a firewall issue.

Tried a few solutions I found online to no avail and finnaly I uninstalled miniconda, restarted my PC and re-installed miniconda.

Went through the procedure of installing spyder through conda.

But again, everything works, except LSP, which keeps crashing/not starting, not able to have the server running. Attached is a debug-log I did. [spyder-debug.log](https://github.com/spyder-ide/spyder/files/7303834/spyder-debug.log)

So, I decided to go back to the original configuration where I installed the standalone Spyder and updated the spyder-kernels to use conda as package manager/interpreter, and it seems that LSP works once again. I do not want to spend more hours in this problem as this is hyper counter-productive.

My non-expert suspicion is that there are some leftover configurations from the standalone install that interfere with the conda install. Afterall, even after uninstalling everything, when I started spyder for the first time, it opened immediately the project I had opened before uninstalling.

Just to inform that I observed this behaviour and that maybe you would like to look into that.

What I did, win10:
Install standalone spyder
Install miniconda->everything fine

Uninstall standalone spyder
Install spyder through conda -> LSP not working

Uninstall miniconda
PC restart
Install miniconda
Install spyder through conda ->LSP still not working (see log)

Install standalone spyder ->LSP working again (conda as interpreter)

**Comment by Jrcarand** [comment]
Hello @ccordoba12 
conda says v4.10.3.0
from spyder I get
* Spyder version: 5.1.5 None
* Python version: 3.7.9 64-bit
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

They were both downloaded 2 days ago, latest stable version available for windows, 64 bit

**Comment by LeaYun** [comment]
@ccordoba12 As for me, resetting works. Thanks!

I installed 5.1.5 but an error reported that ```Your Python environment or installation doesn't have the spyder‑kernels module or the right version of it installed (>= 2.1.0 and < 2.2.0). ``` So I installed the required version by running ```pip install spyder‑kernels==2.1.*``` until all the dependencies were OK. However, the error still remained.

Then I downgraded spyder to 5.0.2. The spyder-kernels started to work but LSP not. Resetting solved the problem.

Now I finally get a chance to sleep.
